include/bamit/sample_functions.hpp

```diff
@@ -0,0 +1,104 @@
+#pragma once
+
+#include <algorithm> // For std::sort
+#include <random> // For std::random_device
+#include <numeric> // For std::reduce
+#include <cmath> // For std::sqrt and std::pow
+
+#include <bamit/Record.hpp>
+#include <bamit/IntervalNode.hpp>
+
+#include <seqan3/io/sam_file/input.hpp>
+#include <seqan3/core/debug_stream.hpp>
+
+namespace bamit
+{
+
+/*! Contains statistics which can be calculated over a sample. */
+struct EstimationResult
+{
+    double mean{0}, median{0}, mode{0}, sd{0}, variance{0};
```

Comment:
Could you add a short description? I was confused by `sd` which stands for standart deviation, right? And I still don't know what `mode` is. But maybe it's just me having trouble with the stochastic vocabulary. :D

Reply:
The mode is simply the value which occurs the most frequently in a list!